### PR TITLE
[ID] Add `Skinning/osu!taiko`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by Nivalyx** [comment]
did a quick look towards this and its original article earlier, and i don't think the `Origin` locations should be translated at all tho since they are considered a technical (programming) variable iirc :

image

if you still want to translate it then you can do sth like `Kiri Bawah (BottomLeft)` or sth like that but yeah the name of the variables should still be mentioned as is somewhere

(i'll try to review this further later tonight/tomorrow if possible)

---

also all instances of `Blend Mode` should be translated to `Mode Blending` and all instances of `Origin` should be translated to something like `Titik Pusat Objek` following graphic design principles iirc

**Comment by Walavouchey** [comment]
> i don't think the Origin locations should be translated at all tho since they are considered a technical (programming) variable fwiw :

disagree; these are meant to be read by skinners, and whether the origin was written in english wouldn't matter to a programmer anyway

File: wiki/Skinning/osu!taiko/id.md [hunk 1]
```diff
@@ -0,0 +1,524 @@
+# Skinning osu!taiko
+
+Kamu dapat mengganti elemen permainan dari osu!taiko dengan menambahkan folder yang bernama `taiko` di dalam folder skin yang digunakan. Jika pengguna tidak melakukan ini, pengguna perlu mengaktifkan fitur ini melalui menu [options](/wiki/Options) (aktifkan opsi `Gunakan skin Taiko untuk mode Taiko`), jika tidak maka skin bawaan yang akan digunakan.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:3]:
```suggestion
Kamu dapat mengganti elemen-elemen permainan osu!taiko dengan menambahkan folder yang bernama `taiko` di dalam folder skin yang digunakan atau dengan mengaktifkan opsi `Gunakan skin Taiko untuk mode Taiko` yang ada pada menu [Options](/wiki/Options). Apabila tidak, osu! akan secara otomatis memasang skin osu!taiko *default* setiap kali kamu bermain osu!taiko.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Ya][true] (lihat catatan) | ![Ya][true] | Normal | Pojok Kiri Bawah | - |
+
+Catatan:
+
+- Nama animasi: `pippidonclear{n}.png`.
+  - Kamu hanya dapat menganimasikan elemen ini dengan maksimum 7 *frame* (dari 0 sampai 6).
+  - Jika dianimasikan, sangat direkomendasikan untuk menganimasikan 7 *frame* tersebut. (Jika tidak, *frame* terakhir dari animasi akan digunakan untuk pengganti *frame* yang tidak dianimasikan dengan urutan *frame* di bawah ini.)
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:19]:
```suggestion
  - Apabila dianimasikan, kamu sangat direkomendasikan untuk menganimasikan ketujuh *frame* tersebut. (Apabila tidak, *frame* terakhir dari animasi akan digunakan untuk pengganti *frame* yang tidak dianimasikan dengan urutan *frame* di bawah ini.)
```

File: wiki/Skinning/osu!taiko/id.md [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+  - Urutan *frame* animasi elemen ini adalah `0 1 2 3 4 5 6 5 6 5 4 3 2 1 0`.
+- Kecepatan animasi tergantung oleh ketukan per menit (BPM).
+- Animasi ini hanya diputar ketika pemain mencapai combo milestone; lalu kembali digantikan elemen idle atau kiai.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:22]:
```suggestion
- Animasi ini hanya akan diputar ketika pemain mencapai angka-angka combo (*combo milestone*) tertentu, yang setelahnya akan kembali digantikan oleh elemen *idle* atau kiai.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Ya][true] (lihat catatan) | ![Ya][true] | Normal | Pojok Kiri Bawah | - |
+
+Catatan:
+
+- Nama animasi: `pippidonfail{n}.png`.
+- Kecepatan animasi tergantung oleh ketukan per menit (BPM).
+- Animasi ini diputar ketika pemain miss selama bermain atau tidak memiliki health yang cukup ketika mencapai break.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:38]:
```suggestion
- Animasi ini akan diputar ketika pemain memperoleh miss pada saat bermain atau tidak memiliki health yang cukup ketika mencapai break.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 5]
```diff
@@ -0,0 +1,524 @@
...
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Ya][true] (lihat catatan) | ![Ya][true] | Normal | Pojok Kiri Bawah | - |
+
+Catatan:
+
+- Nama animasi: `pippidonidle{n}.png`.
+- Kecepatan animasi tergantung oleh ketukan per menit (BPM).
+- Animasi ini diputar ketika pemain tidak berbuat apa-apa (selama break atau menunggu pemain memukul note berikutnya)
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:55]:
```suggestion
- Animasi ini akan diputar ketika pemain sedang tidak aktif bermain (semisal pada saat break atau ketika pemain sedang menunggu note selanjutnya untuk muncul)
```

File: wiki/Skinning/osu!taiko/id.md [hunk 6]
```diff
@@ -0,0 +1,524 @@
...
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Ya][true] (lihat catatan) | ![Ya][true] | Normal | Pojok Kiri Bawah | - |
+
+Catatan:
+
+- Nama animasi: `pippidonkiai{n}.png`.
+- Kecepatan animasi tergantung oleh ketukan per menit (BPM).
+- Animasi ini diputar selama [kiai time](/wiki/Kiai_time).
+- `pippidonfail.png` menggantikan elemen ini jika pemain miss selama kiai time.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:72]:
```suggestion
- `pippidonfail.png` akan menggantikan elemen ini apabila pemain memperoleh miss selama kiai time.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 7]
```diff
@@ -0,0 +1,524 @@
...
+
+![](/wiki/shared/judgement/osu!taiko/taiko-hit300g.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] (lihat catatan) | ![Tidak][false] | Normal | Tengah | - |
+
+Catatan:
+
+- Dapat dianimasikan tetapi hanya *frame* pertama yang digunakan.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:161]:
```suggestion
- Elemen ini dapat dianimasikan, namun hanya *frame* pertama yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 8]
```diff
@@ -0,0 +1,524 @@
...
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Ya][true] | Additive | Tengah | - |
+
+Catatan:
+
+- Diwarnai merah oranye.
+- Elemen ini tidak perlu untuk dibuat untuk osu!taiko.
+  - Elemen ini hanya terlihat ketika taiko bar transparan digunakan.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:282]:
```suggestion
  - Elemen ini hanya akan terlihat apabila taiko bar yang digunakan bersifat transparan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 9]
```diff
@@ -0,0 +1,524 @@
...
+
+![](img/taiko-glow.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Multiplicative | Tengah | 232x232 |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:264]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 10]
```diff
@@ -0,0 +1,524 @@
...
+- Diwarnai kuning.
+- Elemen ini berada di belakang hit position selama [kiai time](/wiki/Kiai_time), elemen ini juga membesar ketika dipukul.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:266]:
```suggestion
- Elemen ini tersembunyi di belakang note-note yang muncul selama [kiai time](/wiki/Kiai_time) dan akan muncul ketika pemain berhasil mengenai note-note tersebut.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 11]
```diff
@@ -0,0 +1,524 @@
...
+
+![](img/taiko-slider.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Ya][true] (lihat catatan) | Normal | Pojok Kanan Atas | 776x162 |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:297]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 12]
```diff
@@ -0,0 +1,524 @@
...
+- Elemen ini berpindah berulang-ulang tak terputus dari arah kanan ke kiri.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:298]:
```suggestion
- Elemen ini akan berpindah secara terus-menerus dari bagian kanan layar menuju ke bagian kiri layar.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 13]
```diff
@@ -0,0 +1,524 @@
...
+- Elemen ini dinonaktifkan jika beatmap memiliki storyboard.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:299]:
```suggestion
- Elemen ini akan dinonaktifkan apabila beatmap yang dimainkan memiliki storyboard.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 14]
```diff
@@ -0,0 +1,524 @@
...
+![](img/taiko-slider-fail.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Tidak][false] | Normal | Pojok Kanan Atas | 776x162 |
+
+Catatan:
+
+- Elemen ini muncul ketika pemain mengalami miss atau health bar tidak mencapai 50% saat memasuki break.
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:315]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 15]
```diff
@@ -0,0 +1,524 @@
...
+
+![](img/taiko-slider-fail.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Tidak][false] | Normal | Pojok Kanan Atas | 776x162 |
+
+Catatan:
+
+- Elemen ini muncul ketika pemain mengalami miss atau health bar tidak mencapai 50% saat memasuki break.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:314]:
```suggestion
- Elemen ini akan muncul pada saat pemain mendapatkan miss atau apabila isi health bar yang ada tidak mencapai 50% pada saat memasuki break.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 16]
```diff
@@ -0,0 +1,524 @@
...
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
+- Elemen ini berpindah berulang-ulang tak terputus dari arah kanan ke kiri.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:316]:
```suggestion
- Elemen ini akan berpindah secara terus-menerus dari bagian kanan layar menuju ke bagian kiri layar.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 17]
```diff
@@ -0,0 +1,524 @@
...
+- Elemen ini dinonaktifkan jika beatmap memiliki storyboard.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:317]:
```suggestion
- Elemen ini akan dinonaktifkan apabila beatmap yang dimainkan memiliki storyboard.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 18]
```diff
@@ -0,0 +1,524 @@
...
+
+![](img/taiko-flower-group.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] (lihat catatan) | ![Ya][true] | Normal | Bawah | - |
+
+Catatan:
+
+- Elemen ini seperti combobursts.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:332]:
```suggestion
- Elemen ini berlaku layaknya comboburst.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 19]
```diff
@@ -0,0 +1,524 @@
...
+- Untuk memiliki comboburst lebih dari satu, gunakan: `taiko-flower-group-{n}.png`.
+  - Salah satu dari *frame* akan muncul ketika pemain mencapai combo milestone.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:334]:
```suggestion
  - Salah satu *frame* akan muncul ketika pemain mencapai angka-angka combo (*combo milestone*) tertentu.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 20]
```diff
@@ -0,0 +1,524 @@
...
+- Elemen ini membesar dan muncul dari belakang pippindon ketika ia berganti status menjadi clear.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:335]:
```suggestion
- Elemen ini akan muncul dari belakang pippidon ketika pemain berhasil mencapai ambang batas clear.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 21]
```diff
@@ -0,0 +1,524 @@
...
+| Semua | ![Tidak][false] | ![Ya][true] | Additive | Tengah | - |
+
+Catatan:
+
+- Diwarnai merah oranye.
+- Elemen ini tidak perlu untuk dibuat untuk osu!taiko.
+  - Elemen ini hanya terlihat ketika taiko bar transparan digunakan.
+- Elemen ini berdenyut di belakang scrolling bar pada hit position saat [kiai time](/wiki/Kiai_time).
+
+## Playfield (separuh atas)
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:285]:
```suggestion
## Playfield (paruh atas)
```

File: wiki/Skinning/osu!taiko/id.md [hunk 22]
```diff
@@ -0,0 +1,524 @@
...
+| Semua | ![Tidak][false] (lihat catatan) | ![Ya][true] | Normal | Bawah | - |
+
+Catatan:
+
+- Elemen ini seperti combobursts.
+- Untuk memiliki comboburst lebih dari satu, gunakan: `taiko-flower-group-{n}.png`.
+  - Salah satu dari *frame* akan muncul ketika pemain mencapai combo milestone.
+- Elemen ini membesar dan muncul dari belakang pippindon ketika ia berganti status menjadi clear.
+
+## Playfield (separuh bawah)
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:337]:
```suggestion
## Playfield (paruh bawah)
```

File: wiki/Skinning/osu!taiko/id.md [hunk 23]
```diff
@@ -0,0 +1,524 @@
...
+
+![](img/taiko-bar-left.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | 181x200 |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:349]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 24]
```diff
@@ -0,0 +1,524 @@
...
+![](img/taiko-drum-inner.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| v1 - v2.0 | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | Max width: 56px |
+| v2.1+ | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | 90x200 |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:366]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 25]
```diff
@@ -0,0 +1,524 @@
...
+![](img/taiko-drum-outer.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| v1 - v2.0 | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | Max width: 72px |
+| v2.1+ | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | 90x200 |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:384]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 26]
```diff
@@ -0,0 +1,524 @@
...
+![](img/taiko-bar-right.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| v1.0 - v2.0 | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | 843x200 |
+| v2.1+ | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | 1024x200 |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:402]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 27]
```diff
@@ -0,0 +1,524 @@
...
+
+![](img/spinner-approachcircle.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Tengah | - |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:518]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 28]
```diff
@@ -0,0 +1,524 @@
...
+
+![](img/spinner-circle.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Tengah | - |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:502]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 29]
```diff
@@ -0,0 +1,524 @@
...
+![](img/taiko-bar-right-glow.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| v1.0 - v2.0 | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | 843x200 |
+| v2.1+ | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | 1024x200 |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:422]:
```suggestion
- Elemen ini ditenggarai dapat menyebabkan *bug-bug* tertentu pada skin yang digunakan.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 30]
```diff
@@ -0,0 +1,524 @@
...
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Tengah | - |
+
+Catatan:
+
+- Elemen ini dicurigai sebagai penyebab bug jika digunakan untuk skin beatmap.
+- Elemen ini juga dipakai mode permainan [osu!](/wiki/Game_mode/osu!).
+- Elemen ini berfungsi sebagai durasi spinner.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:520]:
```suggestion
- Elemen ini berfungsi sebagai indikator durasi spinner.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 31]
```diff
@@ -0,0 +1,524 @@
...
+  - Elemen ini bergerak mengecil.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:521]:
```suggestion
  - Elemen ini akan bergerak mengecil seiring waktu.
```

File: wiki/Skinning/osu!taiko/id.md [hunk 32]
```diff
@@ -0,0 +1,524 @@
...
+
+![](img/spinner-warning.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Blend Mode | Origin | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Semua | ![Tidak][false] | ![Ya][true] | Normal | Tengah | - |
+
+Catatan:
+
+- Elemen ini berfungsi sebagai penanda akan ada spinner.
```
Comment (Nivalyx) [inline wiki/Skinning/osu!taiko/id.md:488]:
```suggestion
- Elemen ini berfungsi sebagai penanda spinner yang akan datang.
```

File: wiki/Skinning/osu!taiko/id.md (hunk 2 above)
Comment (ZeroPyrozen) [reply wiki/Skinning/osu!taiko/id.md:19]:
```suggestion
  - Apabila dianimasikan, kamu sangat direkomendasikan untuk menganimasikan ketujuh *frame* tersebut (Apabila tidak, *frame* terakhir dari animasi akan digunakan untuk menggantikan *frame* yang tidak dianimasikan dengan urutan *frame* di bawah ini).
```
I think this more fit the structure or this one

```suggestion
  - Apabila dianimasikan, kamu sangat direkomendasikan untuk menganimasikan ketujuh *frame* tersebut.
  - Apabila tidak dianimasikan, *frame* terakhir dari animasi akan digunakan untuk menggantikan *frame* yang tidak dianimasikan dengan urutan *frame* di bawah ini.
```

**Comment by ZeroPyrozen** [comment]
About `Origin` and `Blend Mode`, I will change it too in `Skinning/osu!catch` article in separate PR.

File: wiki/Skinning/osu!taiko/id.md (hunk 2 above)
Comment (Nivalyx) [reply wiki/Skinning/osu!taiko/id.md:19]: yeah 👍

**Comment by Nivalyx** [comment]
> > i don't think the Origin locations should be translated at all tho since they are considered a technical (programming) variable fwiw :
> 
> disagree; these are meant to be read by skinners, and whether the origin was written in english wouldn't matter to a programmer anyway

ah i see... the way that they are currently being written (`TopLeft`/`BottomLeft`/etc) made me think that they're some kind of programming variables that shouldn't be translated orz 😦

**Review by Nivalyx — DISMISSED** [review]

File: wiki/Skinning/osu!taiko/id.md [hunk 33]
```diff
@@ -0,0 +1,525 @@
...
+
+---
+
+`taiko-drum-inner.png`
+
+![](img/taiko-drum-inner.png)
+
+| Versi | Dapat Dianimasikan | Dapat Dibuat Sebagai Skin Beatmap | Mode Blending | Titik Pusat Objek | Ukuran Resolusi SD yang Disarankan |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| v1 - v2.0 | ![Tidak][false] | ![Tidak][false] (lihat catatan) | Normal | Pojok Kanan Atas | Max width: 56px |
```
Comment (Walavouchey) [inline wiki/Skinning/osu!taiko/id.md:362]: "Max width" isn't translated

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**